README.ja.md

```diff
@@ -41,7 +41,7 @@ Automates terminal operations.
 
 ## :information_source: 前提条件
 
-cLive は事前に [ttyd](https://tsl0922.github.io/ttyd/) ( バージョン `1.7.0` 以上 ) がインストールされている必要があります。  
```

Comment:
Followed this part.
https://github.com/koki-develop/clive/blob/d40f6db1a54648e7c91b2fec56535ce1fd2c00f4/README.md?plain=1#L44